webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -25,6 +26,8 @@ class NotificationDrawerToggle extends Component {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({
           count: NotificationsStore.getNotifications().length,
+          unreadCount: _.filter(NotificationsStore.getNotifications(),
+            notification => notification.seen).length,
           isLoaded: true
         });
         break;
```

Comment:
i dont think there is a way to test flux connected components.. @gailsteiger  do you know a way to mock the store?